File: POO PHP/Aula12/Cachorro.php
<?php

require_once 'Mamifero.php';

class Cachorro extends Mamifero{

    //Metodos da Classe
    public function enterrarOsso() {
        echo "<p>Enterrando o osso no chao ...</p>";
    }
    public function abanandoRabo() {
        echo "<p>Abanando o rapo pro dono ...</p>";
    }
}

File: POO JAVA/Aula12/src/aula12/Cachorro.java
package aula12;

public class Cachorro extends Mamifero{
    
    //Metodos da Classe
    public void enterrarOsso(){
        System.out.println("Cavando e enterrando osso no chao.");
    }
    public void abanarRabo(){
        System.out.println("Abanando o rabinho pro dono.");
    }
    
    //Metodo de Sobreposição da Classe Mamifero
    @Override
    public void emitirSom() {
        System.out.println("Late Uau, Uau, Uau ....");
    }
}
